docs/web-apps/automated-testing/selenium.md

```diff
@@ -304,35 +294,28 @@ The following example invokes `findElement` on our `driver` instance to locate t
 <TabItem value="Java">
 
 ```java reference title="Finding Selenium WebElements"
-https://github.com/saucelabs-training/demo-java/blob/docs-1.0/selenium-examples/src/test/java/com/saucedemo/selenium/login/SeleniumLoginTest.java#L57-L59
-```
-
-</TabItem>
-<TabItem value="JavaScript">
-
-```js reference title="Finding Selenium WebElements"
-https://github.com/saucelabs-training/demo-js/blob/dac51456e529f952aa9efbd23120439d22c95f14/protractor/appium-web/examples/real-devices/tests/specs/login.spec.js#L12-L14
+https://github.com/saucelabs-training/demo-java/blob/update_to_se4/selenium-examples/src/test/java/com/saucedemo/selenium/login/SeleniumLoginTest.java#L61-L63
```

Comment:
**General comment about code sample refs:** Should we not be using the `/docs-1.0/` tag for these references? I am noticing that in some cases, you reference the main branch, while in others, you reference either `se4` or `update-to-se4` -- are these branches not merged in yet? Do we need to make a note to update these once the training repo is updated, or is that what you meant in your comment about this PR still being a draft because not all languages are merged?

Reply:
Exactly. Code isn't merged into main in all the demo- repos, hoping to get sign off from the rest of the SAs on what I've done. Then merge, then tag, then update this PR, then make it non-draft. :)